bin/resources/GameIndex.yaml

```diff
@@ -26029,7 +26029,7 @@ SLPM-65085:
   name: "Alone in the Dark - The New Nightmare"
   region: "NTSC-J"
 SLPM-65086:
-  name: "Ayumi Hamasaki Dome Tour 2001 - A Visual Mix [Disc1of2]"
+  name: "A Visual Mix - Ayumi Hamasaki Dome Tour 2001 A [Disc1of2]"
```

Comment:
What's with the extra A at the end? is that not at the beginning?

Reply:
Checked on the japanese wiki, it's the official tour name :|

Reply:
Do you have a link?

Reply:
I asked our resident Japanese person

![image](https://user-images.githubusercontent.com/6278726/190407431-1848cd6f-9736-487a-8d86-26e4cd61fa66.png)

Reply:
So the title is "Visual Mix - Ayumi Hamasaki Dome Tour 2001"

Reply:
Sorry it's that one:
https://www.wikiwand.com/ja/%E6%B5%9C%E5%B4%8E%E3%81%82%E3%82%86%E3%81%BF#/%E3%83%A9%E3%82%A4%E3%83%96_/_%E3%82%A4%E3%83%99%E3%83%B3%E3%83%88

Most of her tours had an "A" at the end.

Reply:
I'm pretty sure somebody just romanised her logo because it looks like an A

Reply:
Found her Homepage, it's only a logo.
https://avex.jp/ayu/discography/

Reverted.